src/app/drive/components/FileViewer/FileViewer.tsx

```diff
@@ -11,9 +9,13 @@ import i18n from '../../../i18n/services/i18n.service';
 import { DriveFileData, DriveItemData } from '../../types';
 import { useAppDispatch, useAppSelector } from '../../../store/hooks';
 
+import UilImport from '@iconscout/react-unicons/icons/uil-import';
+import UilMultiply from '@iconscout/react-unicons/icons/uil-multiply';
+
 interface FileViewerProps {
   file: DriveFileData | null;
   onClose: () => void;
+  show: boolean;
```

Comment:
As it seems like `show` prop is used for showing a transition, a better naming could be provided like: `showTransition` or `isLoading`